assignments/arrays.js

```diff
@@ -63,34 +63,44 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
-
-
+const findCarByID = function (id) {
+    // return first match
+    return inventory.filter(car => car.id === id)[0];
+}
+const car33 = findCarByID(33);
+console.log(`Car 33 is a ${car33.car_year} ${car33.car_make} ${car33.car_model}`);
 
 // ==== Challenge 2 ====
 // The dealer needs the information on the last car in their inventory.  What is the make and model of the last car in the inventory?  Log the make and model into the console.
-let lastCar = 0;
-console.log();
+let lastCar = inventory[inventory.length - 1];
+console.log(`The last car in the inventory is a ${lastCar.car_make} ${lastCar.car_model}`);
 
 // ==== Challenge 3 ====
 // The marketing team wants the car models listed alphabetically on the website. Sort all the car model names into alphabetical order and log the results in the console
 let carModels = [];
-console.log();
+inventory.forEach(car => {
+    // is this model already in the array?
+    if (carModels.indexOf(car.car_model) === -1) {
+        carModels.push(car.car_model);
+    }
+});
+console.log(carModels.sort());
 
```

Comment:
Great job. Don't forget that the sort() method is case sensitive, which means that any of the vehicle makes (i.e. the rivolet) that are not capitalized end up out of order. You can fix that by making them all the same case or capitalizing the first letter of each element in your array. Clever idea to check for duplicates! You should also look into the es6 Set method if you want to take that to the next level: 

```suggestion
const carModels = new Set(
  inventory.map(m => (n = m.car_model.toUpperCase())).sort()
);
```